core-web/libs/block-editor/src/lib/AngularRenderer.ts

```diff
@@ -11,18 +11,16 @@ export class AngularRenderer<C, P> {
     private applicationRef: ApplicationRef;
     private componentRef: ComponentRef<C>;
 
-    constructor(component: Type<C>, injector: Injector, props: Partial<P>) {
+    constructor(ViewComponent: Type<C>, injector: Injector, props: Partial<P>) {
         this.applicationRef = injector.get(ApplicationRef);
 
-        const componentFactoryResolver = injector.get(ComponentFactoryResolver);
-        const factory = componentFactoryResolver.resolveComponentFactory(component);
-
-        this.componentRef = factory.create(injector, []);
+        this.componentRef = createComponent(ViewComponent, {
+            environmentInjector: this.applicationRef.injector
+        });
```

Comment:
Why? this is big.

Reply:
@fmontes `componentFactoryResolver.resolveComponentFactory`  is deprecated. The new way to create dynamic components is with the `createComponent`.